apps/condo/domains/contact/components/ContactsEditor/index.tsx

```diff
@@ -353,27 +378,30 @@ export const ContactsEditor: React.FC<IContactEditorProps> = (props) => {
                             )}
                         </Row>
                     </TabPane>
-                    <TabPane
-                        tab={TicketNotFromResidentMessage}
-                        key={CONTACT_EDITOR_TABS.NOT_FROM_RESIDENT}
-                    >
-                        <Row gutter={TAB_PANE_ROW_GUTTERS}>
-                            <Labels
-                                left={PhoneLabel}
-                                right={FullNameLabel}
-                            />
-                            <ContactSyncedAutocompleteFields
-                                initialQuery={initialEmployeesQuery}
-                                refetch={refetchEmployees}
-                                initialValue={!initialValue.id ? initialValue : manuallyTypedContact}
-                                onChange={handleChangeEmployee}
-                                contacts={fetchedEmployees}
-                            />
-                        </Row>
-                    </TabPane>
+                    {
+                        hasNotResidentTab && (
+                            <TabPane
+                                tab={TicketNotFromResidentMessage}
+                                key={CONTACT_TYPE.NOT_RESIDENT}
+                            >
+                                <Row gutter={TAB_PANE_ROW_GUTTERS}>
+                                    <Labels
+                                        left={PhoneLabel}
+                                        right={FullNameLabel}
+                                    />
+                                    <ContactSyncedAutocompleteFields
+                                        initialQuery={initialEmployeesQuery}
+                                        refetch={refetchEmployees}
+                                        initialValue={!initialValue.id ? initialValue : manuallyTypedContact}
```

Comment:
It would be more clear
```suggestion
                                        initialValue={initialValue.id ? manuallyTypedContact : initialValue}
```